client/signup/steps/hosting-decider/hosting-flow-forking-page.tsx

```diff
@@ -25,19 +18,7 @@ export const HostingFlowForkingPage = ( { siteCount }: HostingFlowForkingPagePro
 					alignItems: 'flex-start',
 				},
 			} }
-			title={
-				<span
-					css={ {
-						color: 'var(--studio-gray-100)',
-						fontSize: '3rem',
-						fontWeight: 400,
-						[ MEDIA_QUERIES.small ]: { fontSize: '2rem', textAlign: 'left' },
-						fontFamily: 'Recoleta, sans-serif',
-					} }
-				>
-					{ siteCount === 0 ? __( 'Let’s add your first site' ) : __( 'Let’s add a site' ) }
-				</span>
-			}
+			title=""
```

Comment:
Any reason why we removed the title?

Reply:
Ah I understand now.

In this case, I think we could move away from the `EmptyContent` component and just use a `div`.